lib/job.js

```diff
@@ -86,6 +87,30 @@ class Job extends BaseModel {
     }
 
     /**
+     * Return PR number. Returns null if this is not a PR job
+     * @return {Number}     PR number
+     */
+    get prNum() {
+        delete this.prNum;
```

Comment:
Why is this so heavy-handed?

The logic in the getter doesn't seem that complicated that it needs to be memoized.
```
return this.isPR() ? parseInt(this.name.slice(START_INDEX), 10) : null;
```